calico-enterprise_versioned_docs/version-3.14/getting-started/upgrading/upgrading-calico-to-calico-enterprise/upgrade-to-tsee/standard.mdx

```diff
@@ -4,6 +4,7 @@ description: Steps to upgrade from open source Calico to Calico Enterprise.
 
 # Upgrade from Calico to Calico Enterprise
 
+```javascript
```

Comment:
STET

What happens here from line 7 to line 26 is just some JSX in our MDX file. In most places this kind of stuff happens in components. But here, for some reason, it was added directly. 

In any case, we want this code to stay as it was, here and everywhere else.

Reply:
Fixed.